live-examples/html-examples/image-and-multimedia/map.html

```diff
@@ -0,0 +1,12 @@
+<map name="infographic">
+    <area shape="poly" coords="130,147,200,107,254,219,130,228"
+          href="https://developer.mozilla.org/docs/Web/HTML"
+          target="_blank" alt="HTML" />
+    <area shape="poly" coords="130,147,130,228,6,219,59,107"
+          href="https://developer.mozilla.org/docs/Web/CSS"
+          target="_blank" alt="CSS" />
+    <area shape="poly" coords="130,147,200,107,130,4,59,107"
+          href="https://developer.mozilla.org/docs/Web/JavaScript"
+          target="_blank" alt="JavaScript" />
+</map>
+<img usemap="#infographic" src="/media/examples/mdn-info2.png" alt="MDN infographic">
```

Comment:
Nit: Code style: Explicitly close self closing elements.

Reply:
in `<img>`?

Reply:
Yup, so:

```
<img usemap="#infographic" src="/media/examples/mdn-info2.png" alt="MDN infographic" />
```